tools/rpki-rov.c

```diff
@@ -43,6 +45,18 @@ int connection_error(enum rtr_mgr_status status)
 	return 0;
 }
 
+static int str_to_int(const char *str, int *value) {
+	errno = 0;
+	int tmp = strtol(str, NULL, 10);
+
+	if (errno == 0) {
```

Comment:
recently found a nice post on how to write clean and maintainable `if`s that mainly suggested to rather check for negatives (errors) not positives (as you btw. did below already). Meaning: rewrite this to 
```
if (errno != 0)
        return 1;

*value = tmp;
return 0;
```

Reply:
Done.
Do you have a link? Would love to read more about that.

Reply:
actually was a youtube video: https://www.youtube.com/watch?v=ldqDpmMkXgw